docs/sources/tempo/release-notes/v2-5.md

```diff
@@ -0,0 +1,182 @@
+* Add support querying by `trace:id` and `span:id` [[PR 3670](https://github.com/grafana/tempo/pull/3670)]
+
+### Other enhancements and improvements
+
+* Add TLS support for Memcached Client [[PR 3585](https://github.com/grafana/tempo/pull/3585)]
+* Remove hardcoded delay in distributor shutdown [[PR 3687](https://github.com/grafana/tempo/pull/3687)]
+* Surface new labels for uninstrumented services and systems [[PR 3543](https://github.com/grafana/tempo/pull/3543)]
+* Add messaging-system latency histogram to service-graph [[PR 3453](https://github.com/grafana/tempo/pull/3453)]
+* Add support for sharded ingester queries [[PR 3574](https://github.com/grafana/tempo/pull/3574)]
+* Better compaction throughput and memory usage [[PR 3579](https://github.com/grafana/tempo/pull/3579)]
+* Return a less confusing error message to the client when refusing spans due to ingestion rates. [[PR 3485](https://github.com/grafana/tempo/pull/3485)]
+* Clean Metrics Generator's Prometheus WAL before creating instance [[PR 3548](https://github.com/grafana/tempo/pull/3548)]
+* Delete any remaining objects for empty tenants after a configurable duration, requires config enable [PR 3611](https://github.com/grafana/tempo/pull/3611)]
+
+## Upgrade considerations
+
+When [upgrading](https://grafana.com/docs/tempo/latest/setup/upgrade/) to Tempo 2.5, be aware of these considerations and breaking changes.
+
+### Docker image runs as new UID 10001
+
+The Tempo process in the [official Docker image](https://hub.docker.com/r/grafana/tempo/tags) used to run as `root`. The Tempo process now runs as UID `10001` in the Docker image.
+
+Components such as ingesters and metrics generators that maintain files on disk won't come up cleanly without intervention.
+The new user `10001` won't have access to the old files created by `root`.
+
+The ownership of `/var/tempo` changed from `root:root` to `tempo:tempo` with the UID/GID of `10001`.
+
+The `ingester` and `metrics-generator` statefulsets may need to [run chown](https://opensource.com/article/19/8/linux-chown-command) to change ownership to start properly.
+
+Refer to [PR 2265](https://github.com/grafana/tempo/pull/2265) to see a Jsonnet example of an `init` container.
+
+This change doesn’t impact you if you used the Helm chart with the default security context set in the chart.
+All data should be owned by the `tempo` user already.
+The UID won’t impact Helm chart users.
+
+### Support for vParquet format removed
+
+The original vParquet format [has been removed](https://github.com/grafana/tempo/pull/3663) from Tempo 2.5.
+Direct upgrades from Tempo 2.1 to Tempo 2.5 are not possible.
+You will need to upgrade to an intermediate version and wait for the old vParquet blocks to fall out of retention before upgrading to 2.5. [PR 3663](https://github.com/grafana/tempo/pull/3663)]
+
+vParquet(1) will not be recognized as a valid encoding and cannot read any remaining vParquet blocks.
```

Comment:
vParquet(1) won't be recognized as a valid encoding and any remaining vParquet(1) blocks will not be readable

Reply:
```suggestion
vParquet(1) will not be recognized as a valid encoding and any remaining vParquet(1) blocks won't be readable.
```